tests/test_simple.py

```diff
@@ -3,3 +3,11 @@
 
 def test_simple():
     assert simple_transport(0.25, 1, 1, 0.25) == -0.1875
+
+
+def test_zeros():
+    assert simple_transport(0, 0, 0, 0) == 0
+
+
+def test_conpare():
+    assert simple_transport(0.25, 1, 1, 0.25) > simple_transport(0.5, 1, 1, 0.25)
```

Comment:
```suggestion
    assert simple_transport(0.25, 1, 1, 0.25) > simple_transport(0.5, 1, 1, 0.25)
    
def test_boundary_conditions():
    assert simple_transport(1000, 1e-6, 10, 100) == -999

def test_negative_inputs():
    assert simple_transport(-0.25, 1, -1, -0.25) == 0.3125
```
Added tests to test extremely large or small values and with negative values for parameters.